app/assets/stylesheets/_vocab_sheet.scss

```diff
@@ -1,11 +1,20 @@
 /* Vocab bar background */
 .vocab_sheet_background {
-  background: right top repeat-y;
```

Comment:
would be good to remove this if we can confirm browser support requirements before merging

Reply:
I was able to run this through BrowserStack by the way, looks like we're good for IE 10+ like we saw on caniuse, so that's good unless we need IE <= 9 support.